demos/provenance-demo-slides.html

```
@@ -338,14 +338,14 @@ <h2>Pin the digest, verify the chain</h2>
+    <h2>CycloneDX SBOM, signed and queryable</h2>
     <div class="code">
       <div class="bar"><span class="dot" style="background:#e5534b"></span><span class="dot" style="background:#e3a008"></span><span class="dot" style="background:#76b900"></span><span class="name">cosign verify-attestation + grype</span><button class="copy">copy</button></div>
-<pre><span class="c"># Pull and verify the SPDX SBOM attestation.</span>
-<span class="p">$</span> cosign verify-attestation --type spdxjson \
+<pre><span class="c"># Pull and verify the CycloneDX SBOM attestation (per-platform manifest digest).</span>
+<span class="p">$</span> cosign verify-attestation --type cyclonedx \
     --certificate-oidc-issuer https://token.actions.githubusercontent.com \
-    --certificate-identity-regexp '^https://github\.com/NVIDIA/aicr/\.github/workflows/on-tag\.yaml@refs/tags/.+$' \
-    "$IMAGE@$DIGEST" > predicate.json
+    --certificate-identity-regexp '^https://github\.com/NVIDIA/aicr/\.github/workflows/attest-images\.yaml@refs/tags/.+$' \
+    "$IMAGE@$DIGEST_AMD64" > predicate.json
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)